src/Currency.php

```diff
@@ -187,6 +187,14 @@ final class Currency implements JsonSerializable
             'sub_unit' => 100,
             'sign' => 'P',
         ],
+        'BYR' => [
+            'display_name' => 'Belarussian Ruble',
+            'numeric_code' => 974,
+            'default_fraction_digits' => 0,
+            'sub_unit' => 100,
+            'sign' => 'Br',
+            'deprecated' => true,
```

Comment:
I think we should add this key in all elements.

Reply:
okay